src/onboarding/containers/LoginPageContents.tsx

```diff
@@ -178,11 +180,25 @@ class LoginPageContents extends PureComponent<Props> {
     hasError: errorMessage !== '',
   })
 
-  private handleSubmit = (event: FormEvent) => {
-    const {isValid, errors} = this.validateFieldValues
+  private handleSubmit = async (event: FormEvent) => {
+    event.preventDefault()
     const {email, password} = this.state
 
-    event.preventDefault()
+    if (isFlagEnabled('ssoLogin') && email) {
+      try {
+        const connection = await getConnection(email)
+        if (!!connection) {
+          return this.auth0.authorize({connection})
+        }
+      } catch (e) {
+        const emailError = `${getErrorMessage(
+          e
+        )}. If this issue persists, please contact support@influxdata.com`
+        return this.setState({emailError})
+      }
+    }
```

Comment:
I know this is merged, but on the Quartz side, we ignore errors in this API call and just fall through to the normal login process. We're seeing that the Auth0 API calls fail from time-to-time and we didn't want to break login for the 99.9% of users that aren't using SSO. Up to you if you want to follow that pattern or not.